### ISSUE TITLE
Cannot get access_token

### ISSUE THREAD

**Issue by LarsBuur** [akeneo/pim-community-dev#6618]
**I'm asking a Question**

I am trying to follow your guide from 

```
curl -X POST http://pim.xyz.eu/api/oauth/v1/token \
    -H "Content-Type: application/json" \
    -H "Authorization: Basic XXXXX" \
    -d '{
        "grant_type": "password",
        "username": "xxx",
        "password": "xxx"
    }'
``` 

I keep getting this response 
``` 
{"code":422,"message":"Parameter \"client_id\" is missing or does not match any client, or secret is invalid"}
``` 

Version is 1.7.5 and its a bitnami image. Any clues how to fix this?

**Comment by ahocquard** [comment]
Hello,

First, please check that when you decode the base 64 encoded "client:secret", you get the good client and the secret.

But I think the problem is coming from Bitnami that overrides the Authorization token.

Regards,
Alexandre

**Comment by LarsBuur** [comment]
Thank you for your reply @ahocquard. 

I managed to resolve it - but I have spend an awful long time to figure it out.

It turns out it works if I call curl locally on the server which let me to find this post:  https://www.akeneo.com/forums/topic/code-422-when-trying-to-get-a-token-ce-1-7/

so basically I needed to insert this into some apache conf file.
```
SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
```
after a little digging this was the file I should edit: "/opt/bitnami/apps/akeneo/conf/httpd-app.conf"

and boom.. after a restart of apache now I am getting the access_tokens.
